packages/commerce-sdk-react/src/auth/index.test.ts

```diff
@@ -386,6 +386,15 @@ describe('Auth', () => {
         expect(helpers.loginGuestUser).toHaveBeenCalled()
     })
 
+    test('dnt flag is set correctly', async () => {
+        const auth = new Auth({...config, defaultDnt: true})
+        await auth.loginGuestUser()
+        expect(helpers.loginGuestUser).toHaveBeenCalledWith(
+            expect.anything(),
+            expect.objectContaining({dnt: true})
+        )
+    })
```

Comment:
Could copy/paste and add the `false` / `undefined` condition if you want to error on the safe side in the war against regressions